src/backend/TrafficCourts/Staff.Service/Program.cs

```diff
@@ -99,8 +99,8 @@
 
 app.UseAuthorization();
 
-app.MapControllers()
-    .RequireAuthorization(); // This will set a default policy that says a user has to be authenticated
+app.MapControllers();
+    // .RequireAuthorization(); // This will set a default policy that says a user has to be authenticated
```

Comment:
assume this is temporary and wasn't a mistake commit

Reply:
it is temporary till the pull request for 1213